app/gui2/e2e/edgeInteractions.spec.ts

```diff
@@ -18,7 +18,7 @@ async function initGraph(page: Page) {
 const EDGE_PARTS = 2
 
 /**
-  Scenario: We disconnect the `sum` parameter in the `prod` node by clicking on the edge and pressing the delete key.
+  Scenario: We disconnect the `sum` parameter in the `prod` node by clicking on the edge and clicking on the background.
```

Comment:
The original test doesn't make sense to me. The edge is disconnected by the background-click; the `Delete` after seems spurious.